test/docker-compose.yml

```diff
@@ -102,6 +102,17 @@ services:
       timeout: 10s
       retries: 5
 
+  memcached:
+    image: memcached:alpine
+    user: 11211
```

Comment:
Docker complains about this not being a string. If I wrap it in quotes, it seems to work

Reply:
```suggestion
    user: "11211"
```